java/src/jmri/jmrix/can/cbus/CbusEvent.java

```diff
@@ -105,25 +104,37 @@ public void setEn ( int en ) {
     public void setNn ( int nn ) {
         _nn = nn;
     }
-    
+
     /**
      * Set event name.
      *
-     * @param name Event Name
+     * @param name new Event Name
      */
     public void setName( String name ) {
         _name = name;
     }
-    
+
+    /**
+     * Set event name only if there is no existing name.
+     *
+     * @param name Event Name
+     */
+    public void setNameIfNoName (@Nonnull String name){
```

Comment:
What is this method intended for? It doesn't seem to be used?

Reply:
See https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrix/can/cbus/swing/nodeconfig/CbusNodeRestoreFcuFrame.java#L457